client/platform/web-girder/views/DataShared.vue

```diff
@@ -4,6 +4,7 @@ import {
 } from '@vue/composition-api';
 import type { DataOptions } from 'vuetify';
 import { GirderModel, mixins } from '@girder/components/src';
+import { clientSettings } from 'dive-common/store/settings';
 import { getDatasetList } from '../api';
 import { useStore, LocationType } from '../store/types';
 
```

Comment:
` const tableOptions = reactive({
      page: 1,
      itemsPerPage: 10,
      sortBy: ['created'],
      sortDesc: [true],
    } as DataOptions);`

The default of 10 will likely need to be changed here so that on first load it isn't set to an option that doesn't exist.

Reply:
Looks good!